modules/common/html-utils/src/main/resources/xml/xslt/html5-outliner.xsl

```diff
@@ -225,19 +227,19 @@
                 <xsl:sequence select="f:heading-content($node/*[f:rank(.)=$rank][1],$parent)"/>
             </xsl:when>
             <xsl:when test="$parent[self::body]">
-                <xsl:sequence select="'Untitled document'"/>
```

Comment:
Not really sure why I changed this either. Seems unnecessary.

Reply:
This code has now moved to https://github.com/daisy/pipeline-modules/blob/3404b2828e70c2eab47c26d4e5e5b4de7248fdcb/common/html-utils/src/main/resources/xml/xslt/untitled-section-titles.xml (which also uses text nodes instead of strings).